### DISCUSSION TITLE
可以添加鼠标点击效果吗?

### DISCUSSION THREAD

**Discussion by Caodaping** [shuax/MouseInc.Settings#146] [Ideas]
现在的屏幕越来越大，27寸，32寸的显示器也非常常见。
窗口开多了以后，有时候忽然找不到鼠标在哪里，需要使劲地晃几下鼠标才能找到。

MouseInc能否添加一些鼠标点击效果，便于快速找到光标呢? 我们使用录制屏幕的软件的时候，时常看到它具有类似功能。
比如连续点击三次以上的鼠标时，有一个光标为中心的圆圈，逐渐地放大，像水波纹一样。左中右键设置不同的颜色、线宽、圆圈大小等........

这里有示例效果(但是这个是网页，而且太花哨，不需要如此复杂)：[https://perphet.com/2017/12/%E7%BB%99%E7%BD%91%E9%A1%B5%E6%B7%BB%E5%8A%A0%E9%BC%A0%E6%A0%87%E7%82%B9%E5%87%BB%E7%89%B9%E6%95%88/](https://perphet.com/2017/12/%E7%BB%99%E7%BD%91%E9%A1%B5%E6%B7%BB%E5%8A%A0%E9%BC%A0%E6%A0%87%E7%82%B9%E5%87%BB%E7%89%B9%E6%95%88/)

谢谢作者!!

**Comment by shuax** [comment]
可以考虑，暂时不会加
